### DISCUSSION TITLE
How to debug Telescope live grep (find text in files) hangs everytime?

### DISCUSSION THREAD

**Discussion by lsshawn** [LazyVim/LazyVim#265] [Q&A]
It hangs even for a small project.

Do I disable plugins one by one to find  the cause?

**Comment by folke** [comment]
Do you have ripgrep installed? Either way, better to ask in the telescope repo

**Reply by lsshawn** [reply]
Yeah I have it installed. Was working fine a few days ago. Will ask in telescope repo.
